### PR TITLE
Add console config

### PR DESCRIPTION
**Pull request by 2004-okari**
In this pull request  I have added core functionality to your console app, ensuring it behaves as expected and follows the specified requirements. Here's an overview of the changes:

**`app.rb` - Console App Entry-Point:**
- Created an `app.rb` file to serve as the entry-point for the console app.
- Implemented methods to perform the following actions:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain Person).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person ID.

**`main.rb` - Entry Point Method:**
- Defined an entry point method called `main` within `main.rb`. This method is invoked at the end of the file.
- Implemented functionality to:
  - Present the user with a list of options to perform.
  - Allow users to choose an option.
  - Prompt for parameters when needed.
  - Provide a way to quit the app gracefully.

With these enhancements, your console app now provides a user-friendly interface and allows users to interact with it seamlessly. Users can perform various actions related to books, people, and rentals, making it a versatile tool.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+require_relative 'person'
```
Comment (Alfredbis29) [inline app.rb:1]: - [x] I'm sorry I was not able to inline in your teacher.rb but I've reviewed your code and found that in `teacher. rb`, the initialize method expects 4 arguments but was given only 3. To fix this, update the method signature to accept 4 parameters and ensure they are used correctly within the method. Don't forget to validate input data and provide meaningful parameter names for clarity.
Screenshot from 2023-10-06 12-12-42

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    when 't'
+      puts "Enter teacher's specialization:"
+      specialization = gets.chomp
+
+      person = Teacher.new(name, age, specialization)
+    when 's'
+      puts "Enter student's classroom:"
+      classroom = gets.chomp
+
+      person = Student.new(name, age, classroom)
```
Comment (Alfredbis29) [inline app.rb]: - [ ] Your effort on this project is commendable; however, there's a crucial issue: creating a person with complete credentials is not functioning correctly. Please address this to ensure the project meets all required functionalities. I can see the that you do have the option to `Enter student's class room`  I got confused on the way to complicate this section while I can't see the logic you applied here , and also the parent permission is not displaying , as shown in the video below follow how the parameter and options are arranged to make your project look professional
[Check this video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @2004-okari  
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark:  Nice work so far :+1: 
 
 ## Required Changes 

- [ ] Check Under the review 

 ## Optional suggestions 

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @2004-okari                   👋

Your project is complete! ✔️✔  🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors


- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**